content/Google is a modal dialog - Martin Rubi.md

```diff
@@ -0,0 +1,116 @@
+to solve a specific task hidding every other complexity not related to that task.
+
+Once the user ends solving that specific part of the app it returns to the previous
+flow just like she left it.
+
+Whichs is pretty much the same workflow that we use to google things
+
+### Google is a modal dialog
+
+How do we google things?
+
+Usually we are doing something like writing a blog post and we find the need to
+search for information.
+
+We open a new tab with a clean and very specific design to do just one thing,
+search things.
+
+We select a few results, decide whether our question was answered or not and get
+back to our previous task closing the search tab.
+
+That is what modal dialogs functional purpose is regardless of its implementation
+on the user interface space.
+
+### Other issues with edit in place
+
+There are two very important functions in a modal dialog.
+
+The first one would be the one already mentioned, to provide an isolated context
+to solve a specific task hidding all other context not relevant to that task.
+
+Yet there is another function just as important: hidding unnecessary complexity
+in the main application interface until that complexity is required.
+
+The rationale of a modal dialog is to present to the user the complexity of
+a user interface when the user needs it, not before, not afterwards, and just for
+the time the user is dealing with that task.
+
+The proliferation of menu options and visual aids can hurt more than they would help
+producing a sense of overwhelming and confusion in the user.
+
+One example of that value are context menus where instead of always showing all options
+to the user they show just the options for a particular topic and only when the user
+requests it, for example after a right button click.
+
+Popup context menus are modal dialogs. So are tooltips.
+
+Edit in place has other problems that modal dialogs solve in a clear and explicit
+way like commiting or rolling back an edition.
+
+A modal dialog requires an explicit action to accept an edition and provides a clear
+interface to dismiss the edition in case the user wants to roll it back.
+
+Edit in place has ctl-z but that produces very different experiencies in the user: when is
+a change applied in a edit in place field, when it loses focus, on every char typed,
+when Enter is hit? Do I have to hit Enter or would that trigger a different action like
+a form submit? Where would an Enter hit leave the focus? What if my change brakes
+something in real time? Can I have a preview of the edition result before applying it for good?
+
+All those questions are explicitly answered in a modal dialog interface.
+
+### Small devices can use modals too
+
+In smaller devices edit in place can be a nightmare for the ones that find it hard to
+hit the proper field or button. It's a very common source of frustration to tap the
+almost correct yet not exactly that one field.
```

Comment:
WDYT about `tap the "almost correct yet not exactly that one" field.` to highlight that phrase as a unit?